### PR TITLE
Rk 160898486 add fuel surcharge placeholder

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Description

Adds in a stub function for the calculation of fuel surcharge inside rate engine, adds this value to the shipment_line_items table, and adjusts EDI generator to use this field in the L1 segment.

## Setup
```sh
make db_dev_migrate 
```
to run the migration on your local dev db.

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](./docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](./docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](./docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
* [ ] There are no aXe warnings for UI.
* [ ] This works in IE.
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @willowbl00
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2136865/stories/160898486) for this change

### CODE REVIEW

File: migrations/20181217162650_add_fuel_surcharge.up.fizz [hunk 1]
```diff
@@ -0,0 +1 @@
+add_column("shipment_line_items", "fuel_surcharge", "int", {})
```
Comment (kimallen) [inline migrations/20181217162650_add_fuel_surcharge.up.fizz]: We don't need to add a new column for the fuel surcharge.  The calculated value from the rate engine should just go under amount_cents.  If there is a single rate value we can extract from the rate engine, that value will go under applied_rate

Comment (Ryan-Koch) [reply migrations/20181217162650_add_fuel_surcharge.up.fizz]: Understood. I'll get rid of the migration and redirect my effort that way and leave another note once there's a commit with that done. Thanks!

File: pkg/rateengine/linehaul_test.go [hunk 2]
```diff
@@ -203,3 +203,11 @@ func (suite *RateEngineSuite) Test_CheckLinehaulChargeTotal() {
 		t.Errorf("Determined linehaul factor incorrectly. Expected %d, got %d", expected, cost.LinehaulChargeTotal)
 	}
 }
+
+// TODO: Once the fuel surcharge calculation is in place, add in a proper test for it.
+func (suite *RateEngineSuite) Test_CheckFuelSurchargeComputation() {
+	engine := NewRateEngine(suite.db, suite.logger, suite.planner)
+	fuelSurcharge, err := engine.fuelSurchargeComputation()
```
[the remark this answers is not in the document]
Comment (Ryan-Koch) [reply pkg/rateengine/linehaul_test.go]: Can do!

**Comment by Ryan-Koch** [comment]
@kimallen changes from our pairing session and another small change to fix a broken test have been pushed.

**Comment by kimallen** [comment]
Looks good to me!

**Pull request merged**